src/tests/test_module.py

```diff
@@ -537,6 +537,42 @@ def test_agg_avg(self):
             actual_result = r.execute_command('TS.RANGE', agg_key, 10, 50)
             assert expected_result == actual_result
 
+    def test_agg_std_p(self):
+        with self.redis() as r:
+            agg_key = self._insert_agg_data(r, 'tester', 'std.p')
```

Comment:
Too much hard coded stuff in the test, which will make it very flaky.
I recommend defining a list with values and inserting it into the redis, then computing the std using python standard library and the implementation you did and assert they have similar values.